core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.properties

```
@@ -1,3 +1,4 @@
+Disable\ remember\ me=Disable "Keep me signed in"
```

This is what I meant when I wrote typographic quotes:

```suggestion
Disable\ remember\ me=Disable “Keep me signed in”
```

---

Thanks. I didn't know the difference. 

Silly question: should I use typographic quotes in Javadoc? I probably shouldn't right?

---

No, but it's preferable for regular UI text.